lib/elixir/test/elixir/task/supervisor_test.exs

```diff
@@ -339,7 +339,8 @@ defmodule Task.SupervisorTest do
                [ok: other_supervisor, ok: supervisor, ok: other_supervisor, ok: supervisor]
     end
 
-    test "streams an enumerable with fun and executes supervisor fun in monitor process", context do
+    test "streams an enumerable with fun and executes supervisor fun in monitor process",
```

Comment:
This does not change formatting for me with 

```bash
mix format lib/elixir/test/elixir/task/supervisor_test.exs
```

🤔

Reply:
`mix format --check` on travis was failing, so I ran mix format again and it gave me this... Maybe it's changed on master, I'll rebase and try again.